misc/config_tools/data/generic_board/hybrid.xml

```diff
@@ -119,7 +120,7 @@
   </vm>
   <vm id="1">
     <vm_type>SERVICE_VM</vm_type>
-    <name>ACRN Service VM</name>
+    <name>ACRN_Service_VM</name>
```

Comment:
Do we need to add the underscores characters?

Reply:
There the whitespace is not allowed, we can add any characters except whitespace to replace it.